core/org.eclipse.birt.core/src/org/eclipse/birt/core/script/JavascriptEvalUtil.java

```diff
@@ -386,4 +390,18 @@ private static Object getSecurityDomain(final String file) {
 		}
 	}
 
+	/**
+	 * Evaluate the system property to use the JavaScript security based on
+	 * certificates
+	 */
+	private static boolean isECMAScriptSecurityEnabled() {
+		boolean scriptSecurity = false;
+		/* System property: -Dbirt.ecmascript.security */
+		String configuredEcmaScriptSecurityProperty = System.getProperty(ECMA_SCRIPT_SECURITY_PROPERTY_KEY);
+		if (configuredEcmaScriptSecurityProperty != null
+				&& configuredEcmaScriptSecurityProperty.equalsIgnoreCase(ECMA_SCRIPT_SECURITY_ENABLED)) {
+			scriptSecurity = true;
+		}
+		return scriptSecurity;
+	}
```

Comment:
While this is fully functional, I would probably go for "true" / "false" here instead ( and alter the name of the property to something like: "birt.ecmascript.enable-security"). This would potentially reduce the code to:

return Boolean.getBoolean(ECMA_SCRIPT_ENABLE_SECURITY_PROPERTY_KEY); // getBoolean actually gets the value from System.getProperty()

Reply:
Is changed.